framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -207,7 +207,7 @@ public abstract class GenericAnnotatedTypeFactory<
      *
      * @see #getAnnotatedTypeLhs(Tree)
      */
-    private boolean useFlow;
+    public boolean useFlow;
```

Comment:
Would `protected` work? I had that as comment on https://github.com/opprop/checker-framework/pull/215, but just notice that I didn't publish that comment...

Reply:
I think it will, as all the XXXAnnotatedTypeFactory are subclasses of `GenericAnnotatedTypeFactory.java`